sdk/search/search-documents/test/odata.spec.ts

```diff
@@ -30,6 +30,73 @@ describe("odata", () => {
     assert.strictEqual(result, "search.ismatch('you''re', 'Description')");
   });
 
+  it("no arguments", () => {
+    assert.strictEqual(odata`Foo eq 2`, "Foo eq 2");
+  });
+
+  it("one argument", () => {
+    assert.strictEqual(odata`Foo eq ${2}`, "Foo eq 2");
+  });
+
+  it("many arguments", () => {
+    assert.strictEqual(odata`Foo eq ${2} and Bar eq ${3}`, "Foo eq 2 and Bar eq 3");
+    assert.strictEqual(
+      odata`Foo eq ${2} and Bar eq ${3} and Baz eq ${4}`,
+      "Foo eq 2 and Bar eq 3 and Baz eq 4"
+    );
+    assert.strictEqual(
+      odata`Foo eq ${2} and Bar eq ${3} and Baz eq ${4} and Qux eq ${5}`,
+      "Foo eq 2 and Bar eq 3 and Baz eq 4 and Qux eq 5"
+    );
+    assert.strictEqual(
+      odata`Foo eq ${2} and Bar eq ${3} and Baz eq ${4} and Qux eq ${5} and Quux eq ${6}`,
+      "Foo eq 2 and Bar eq 3 and Baz eq 4 and Qux eq 5 and Quux eq 6"
+    );
+  });
+
+  it("null", () => {
+    assert.strictEqual(odata`Foo eq ${null}`, "Foo eq null");
+  });
+
+  it("bool", () => {
+    let x: boolean = true;
+    assert.strictEqual(odata`Foo eq ${x}`, "Foo eq true");
+    assert.strictEqual(odata`Foo eq ${true}`, "Foo eq true");
+
+    x = false;
+    assert.strictEqual(odata`Foo eq ${x}`, "Foo eq false");
+    assert.strictEqual(odata`Foo eq ${false}`, "Foo eq false");
```

Comment:
Again this feels too exhaustive. I think checking with a singular boolean value is sufficient. We don't need to prove the language feature works, just that our helper does.

Reply:
Removed it